cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLSink.java

```diff
@@ -122,5 +105,10 @@ public String getTransactionIsolationLevel() {
       return ImmutableMap.of(
           CloudSQLMySQLConstants.CONNECTION_TIMEOUT, String.valueOf(connectionTimeout));
     }
+
+    @Override
+    protected CloudSQLMySQLConnectorConfig getConnection() {
```

Comment:
Is this nullable? The connection won't have value at deploy time

Reply:
ack, added annotation.